src/libsync/theme.cpp

```diff
@@ -139,16 +144,26 @@ QIcon Theme::themeIcon(const QString &name, bool sysTray) const
             return cached = QIcon::fromTheme(name);
         }
 
-        QList<int> sizes;
-        sizes << 16 << 32 << 64 << 128 << 256;
-        foreach (int size, sizes) {
-            QString svgName = QString::fromLatin1(":/client/theme/%1/%2.svg").arg(flavor).arg(name);
-            QSvgRenderer renderer(svgName);
+        const auto svgName = QString::fromLatin1(":/client/theme/%1/%2.svg").arg(flavor).arg(name);
+        QSvgRenderer renderer(svgName);
+        const auto createPixmapFromSvg = [&renderer] (int size) {
             QImage img(size, size, QImage::Format_ARGB32);
             img.fill(Qt::GlobalColor::transparent);
             QPainter imgPainter(&img);
             renderer.render(&imgPainter);
-            auto px = QPixmap::fromImage(img);
+            return QPixmap::fromImage(img);
+        };
+
+        const auto sizes = isBranded() ? QVector<int>{ 16, 22, 32, 48, 64, 128, 256, 512, 1024 }
```

Comment:
I guess there is a non-technical reason why branded versions use other size variants?

Reply:
Legacy reasons mostly, we can have assets on any of those sizes.